### DISCUSSION TITLE
Not able to deploy a podman container .

### DISCUSSION THREAD

**Discussion by Mohit-orange** [containers/podman#23885] [General]
Hello Guys,

I am trying to deploy a podman container image but my deployment is getting failed with error Error: ImagePullBackOff.

Below is the detail.

![image](https://github.com/user-attachments/assets/0e01f616-cdfd-4ca7-9038-db24e5bb4dc9)

azureuser@Master:~$ sudo kubectl describe pod -n netbackup

![image](https://github.com/user-attachments/assets/adc0aab8-b2f6-44f9-a7f6-b82c9c84fd1e)

I have checked and found, I can access the repo with credentials.
![image](https://github.com/user-attachments/assets/328aae2e-14ea-4dd1-bfd8-1c2a2a4c78f8)

I can manually pull the same image from my repository on my master/control-pane as well as from both worker nodes.
![image](https://github.com/user-attachments/assets/efa1f105-12f7-4b0f-bcb9-46bfffa2a791)

![image](https://github.com/user-attachments/assets/5c51c6da-5dbf-4a78-849a-557cf343ffec)

![image](https://github.com/user-attachments/assets/0905724e-a122-4793-8d62-ca76879fc95b)

azureuser@Master:$ helm version
version.BuildInfo{Version:"v3.15.4", GitCommit:"fa9efb07d9d8debbb4306d72af76a383895aa8c4", GitTreeState:"clean", GoVersion:"go1.22.6"}
azureuser@Master:$
azureuser@Master:$ sudo kubectl version
Client Version: v1.30.0
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.30.0
azureuser@Master:$
azureuser@Master:$ kubectl version --client
Client Version: v1.30.0
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
azureuser@Master:$
azureuser@Master:$ podman version
Version: 3.4.4
API Version: 3.4.4
Go Version: go1.18.1
Built: Thu Jan 1 00:00:00 1970
OS/Arch: linux/amd64
azureuser@Master:$
azureuser@Master:~$ docker version
Emulate Docker CLI using podman. Create /etc/containers/nodocker to quiet msg.
Version: 3.4.4
API Version: 3.4.4
Go Version: go1.18.1
Built: Thu Jan 1 00:00:00 1970
OS/Arch: linux/amd64
azureuser@Master:$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.4 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.4 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
azureuser@Master:$

![image](https://github.com/user-attachments/assets/873d005a-7f76-4a81-a0cb-07f3e50ab29e)

![image](https://github.com/user-attachments/assets/2faf2215-badf-44cd-82b9-87cdf7e2458b)

**Comment by eriksjolund** [comment]
In the last screenshot there is the text 

```
Failed to pull image "myprivateregistry.local:5000/nbk8splugin:latest": reading manifest latest in myprivateregistry.local:5000/nbk8splugin: unauthorized: authentication required
```

I guess problem is probably related to _unauthorized: authentication required_ 

Side note: please don't use screenshots for terminal content, instead use markdown formatting. 
_To format code or text into its own distinct block, use triple backticks._
quote from https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code
